config/development.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     id:                   'aws-provisioner2-dev',
     workerTypeTableName:  'AwsWorkerTypesDev2',
     secretTableName:      'ProvisionerSecretsDev',
+    workerStateTableName: 'ProvisionerWorkerStateDev1',
```

Comment:
because i was trying to figure out an issue and screwed up the non-1 table... it's not as easy to delete tables as it is to make them.